README.md

```diff
@@ -69,7 +71,7 @@ extern crate alloc_system;
 
 * Build the project (in release mode to optimize the size of the binary)
 ```
-cargo build --release
+cargo build --release --target=mipsel-unknown-linux-gnu
```

Comment:
Did you add this just for clarity? I don't think I needed this but maybe my setup was just different.